contracts/staking/contracts/src/immutable/MixinDeploymentConstants.sol

```diff
@@ -50,6 +52,10 @@ contract MixinDeploymentConstants {
     // Ropsten & Rinkeby Weth Asset Data
     // bytes constant internal WETH_ASSET_DATA = hex"f47261b0000000000000000000000000c778417e063141139fce010982780140aa0cd5ab";
 
+    // @TODO SET THESE VALUES FOR DEPLOYMENT
+    address constant public WETH_ASSET_PROXY_ADDRESS = address(1);
```

Comment:
What would you think about making these constants `private` and depending solely on the getters?

This would make it very easy to override for tests (and for different networks).

Reply:
We could also get rid of all these constants that have been commented out.

Reply:
I like that. I'll leave the commented out ones in for testnet deployments for now.